__tests__/auth.spec.js

```diff
@@ -0,0 +1,85 @@
+import jsonwebtoken from "jsonwebtoken";
+import * as auth from "../auth/auth.js";
+
+function InvalidHeadersReq() {
+    const req = new Object();
```

Comment:
```suggestion
    const req = {};
```
Apparently this is a more efficient way of creating empty objects.